### PR TITLE
StaticHtmlReporter: Make sure projects are listed first

### PR DESCRIPTION
**Pull request by mnonnenmacher**
Make sure that in the dependency table the project itself is always the
first entry.

### CODE REVIEW

File: reporter/src/funTest/assets/static-html-reporter-test-expected-output.html [hunk 1]
```diff
@@ -424,7 +424,7 @@
 </script>
     <div id="report-container">
       <div class="ort-report-label">Scan Report</div>
-      <div>Created by <strong>ORT</strong>, the <a href="http://oss-review-toolkit.org/">OSS Review Toolkit</a>, version <REPLACE_ORT_VERSION> on <REPLACE_TIMESTAMP>.</div>
+      <div>Created by <strong>ORT</strong>, the <a href="http://oss-review-toolkit.org/">OSS Review Toolkit</a>, version 0.1.0-beta.0.4261+20191121T133123Z on <REPLACE_TIMESTAMP>.</div>
```
Comment (sschuberth) [inline reporter/src/funTest/assets/static-html-reporter-test-expected-output.html]: This also still needs `<REPLACE_ORT_VERSION>`.

File: reporter/src/funTest/assets/static-html-reporter-test-input.yml [hunk 2]
```diff
@@ -67,7 +67,7 @@ analyzer:
           - id: "Maven:org.apache.commons:commons-lang3:3.5"
       - name: "testCompile"
         dependencies:
-        - id: "Maven:junit:junit:4.12"
+        - id: "Ant:junit:junit:4.12"
```
Comment (sschuberth) [inline reporter/src/funTest/assets/static-html-reporter-test-input.yml]: Why was this changed to `Ant`?

File: reporter/src/funTest/assets/static-html-reporter-test-expected-output.html (hunk 1 above)
Comment (mnonnenmacher) [reply reporter/src/funTest/assets/static-html-reporter-test-expected-output.html]: Thanks, done.

File: reporter/src/funTest/assets/static-html-reporter-test-input.yml (hunk 2 above)
Comment (mnonnenmacher) [reply reporter/src/funTest/assets/static-html-reporter-test-input.yml]: To have a dependency that would appear before the project without this change.
